nixos/tests/podman-dnsname.nix

```diff
@@ -0,0 +1,42 @@
+import ./make-test-python.nix (
```

Comment:
As mentioned, I'll happily merge this into the `podman` test after #123841 (unless you want it to be separate)